googleMapsApiModule.py

```diff
@@ -4,27 +4,24 @@
 import mysql.connector
 from constants import BASE_URL
 
-gmaps = googlemaps.Client(config.key)
-#global variable gmaps
+gmaps = googlemaps.Client(config.key)      #global variable gmaps
 
 def direction(origin,destination):
     result = gmaps.directions(origin,destination)
     address = "origin="+origin+"&"+"destination="+destination
-    address = address.lower()
-    address = address.replace(" ","+")
-    result_url = "{}&{}".format(BASE_URL['direction'], address.lower().replace(" ", "+"))
+    url = BASE_URL['direction']
+    result_url = f'{direction}&{address.lower().replace(" ", "+")}'
     print(result_url)
     webbrowser.open_new(result_url)
 
 
 def geocoding(search_location):
-    gmaps = googlemaps.Client(config.key)
     result = gmaps.geocode(search_location)
     print("Formatted Address: "+result[0]['formatted_address'])
     print("Latitude: "+str(result[0]['geometry']['location']['lat'])+" "+"Longitude: "+str(result[0]['geometry']['location']['lng']))
-    address = search_location.lower()
-    address = address.replace(" ","+")
-    result_url = "{}={}".format(BASE_URL['geocoding'], address.lower().replace(" ", "+"))
+    address = search_location
+    url = BASE_URL['geocoding']
+    result_url = f'{url}={address.lower().replace(" ", "+")}'
     webbrowser.open_new(result_url)
 
 def mapsstatic(search_location):
```

Comment:
Use fstring in the url generation of this method too

Reply:
@vishakha-lall are you referring to the mapsstatic function?

Reply:
Yes